examples/JsEventHandlerExamples/actionBar-back-to-list.js

```diff
@@ -0,0 +1,130 @@
+﻿/**
+ * Name: Relativity Back To List Action Bar Event Handler
+ * Description: This event handler adds a "Back to List" button to the action bar that navigates the user to the List Page of the current active `ArtifactTypeId`.
+ */
+(function(eventNames, convenienceApi) {
+    const APP_ID = "appid";
+    const ARTIFACT_TYPE_ID = "artifacttypeid";
+    const PARENT_ARTIFACT_ID = "parentartifactid";
+    const LIST_PAGE_MODE = "ListPage";
+    const RELATIVITY_INTERNAL_PAGE = "relativityinternal.aspx";
+    const FORMS_IFRAME = "fdfeeecf-449d-4c86-8c10-b062f58020c5/index.html";
+    const BACK_TO_LIST_BUTTON_ID = "backToListButton";
+
+    /**
+    * @desc returns corresponding List Page URL
+    * @returns {String|undefined} the URL to redirect to
+    */
+    function generateBackListPageUrl() {
+        const currentUrl = new URL(window.location.href);
```

Comment:
Is it worth documenting that there may be more preferred ways of getting the URL in R1, and link to some PageBase docs?